### DISCUSSION TITLE
A rule to enforce `readonly` modifier in typescript class

### DISCUSSION THREAD

**Discussion by freiondrej-lokalise** [biomejs/biome#4310] [Q&A] [answered]
Hello, I found https://biomejs.dev/linter/rules/use-naming-convention/ which does work with the modifiers, but not quite how I thought it would - I hoped it could automatically apply the readonly modifier to members which aren't written to outside the constructor.

**Comment by Conaclos** [comment, marked as answer]
This is not the purpose of [useNamingConvention](https://biomejs.dev/linter/rules/use-naming-convention/). Please add comment to #3 to request a new rule and possibly a link to an existing rule of another linter that does what you want.

**Reply by freiondrej-lokalise** [reply]
Thanks for pointing me in the right direction! 🙇 created https://github.com/biomejs/biome/discussions/3#discussioncomment-10982476 .
